_posts/2023-05-16-cockpit-292.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Cockpit 292
+author: pitti
+date: '2023-05-16'
+tags: cockpit, machines
+slug: cockpit-292
+category: release
+summary: 'Per-service disk IO, right-to-left language fixes, robust VM watchdog changes'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 292 and cockpit-machines 290:
+
+
+## Metrics: Add disk I/O per service
+
+The "Disks" usage metrics card now shows the active read and write bandwidth for the top 5 consumers by systemd unit. This is similar to the already existing per-service CPU and RAM usage.
+
+![metrics-disks-io-cgroups]({{ site.baseurl }}/images/292-metrics-add-disk-io-per-service.png)
+
+## Machines: More robust watchdog addition
+
+In some cases, adding a watchdog to a running VM fails. When that happens, the changes can only be applied through rebooting the VM. The "Add watchdog" dialog now supports this case.
+
+![234253257-d6bc8d91-7b66-4ef7-b9df-79d554f92b0b]({{ site.baseurl }}/images/292-machines-apply-watchdog-changes-on-next-boot-if-hotplug-fails.png)
+## Several right-to-left language fixes
+
+In [Cockpit 282](https://cockpit-project.org/blog/cockpit-282.html), the first round of RTL support arrived. This release of Cockpit further dramatically improves RTL support.
+
+_Huge thanks to @ahmedr2001 for kickstarting this most recent round of improvements!_
```

Comment:
`s/Huge/A big/`

This is more about adding the "a" than changing "huge" to "big" — it's just that when i read "A huge" in my head, it sounds bad :sweat_smile: 

Do we want to link to the username on GitHub?